.github/workflows/examples.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         toolchain:
           - stable
         job:
-          - build
+          - contract build
```

Comment:
Is this just a label? Or is this a mistake? I think for `build` we still want to use `cargo contract`.

Reply:
this is a mistake indeed, thanks for noting

Reply:
> Is this just a label? Or is this a mistake? I think for `build` we still want to use `cargo contract`.

No, this is not a mistake. Later down the stage, the job label is concatenated with `cargo` command.
So this way `contract build` job label will be concatenated into `cargo contract build` and `test` label will respectively become just `cargo test`